flash/data/data_module.py

```diff
@@ -164,25 +167,35 @@ def _show_batch(self, stage: RunningStage, reset: bool = True) -> None:
                 iter_dataloader = self._reset_iterator(stage)
                 _ = next(iter_dataloader)
             data_fetcher: BaseVisualization = self.data_fetcher
+            # list of functions to visualise
+            data_fetcher._fcn_white_list = self._fcn_white_list
             data_fetcher._show(stage)
             if reset:
                 self.viz.batches[stage] = {}
 
-    def show_train_batch(self, reset: bool = True) -> None:
+    def show_train_batch(self, name: str = 'load_sample', reset: bool = True) -> None:
```

Comment:
List of hook names should also be supported.

Reply:
I can provide the following api:
```python
def show_train_batch(self, name: Union[str, List[str]] = 'load_sample', reset: bool = True) -> None:
    ...
```